pgsql/pc_inout.c

```diff
@@ -32,6 +32,7 @@ Datum pcpoint_as_text(PG_FUNCTION_ARGS);
 Datum pcpatch_as_text(PG_FUNCTION_ARGS);
 Datum pcpoint_as_bytea(PG_FUNCTION_ARGS);
 Datum pcpatch_bytea_envelope(PG_FUNCTION_ARGS);
+Datum pcpatch_bytea_bounding_diagonal(PG_FUNCTION_ARGS);
```

Comment:
I would prefer `pcpatch_bounding_diagonal_as_bytea`. (I propose to rename the `pcpatch_bytea_envelope` to `pcpatch_envelope_as_bytea` in my pc_envelope PR.